import-export-cli/integration/base/helper.go

```diff
@@ -292,3 +295,46 @@ func CountFiles(path string) (int, error) {
 	}
 	return i, nil
 }
+
+// IsFileAvailable checks if a file exists and is not a directory before we
+// try using it to prevent further errors.
+func IsFileAvailable(filename string) bool {
+	info, err := os.Stat(filename)
+	if os.IsNotExist(err) {
+		return false
+	}
+	return !info.IsDir()
+}
+
+// Copy the src file to dst. Any existing file will be overwritten
+func Copy(src, dst string) error {
+	in, err := os.Open(src)
+	if err != nil {
+		return err
+	}
+	defer in.Close()
+
+	out, err := os.Create(dst)
+	if err != nil {
+		fmt.Println(err)
+		return err
+	}
+	defer out.Close()
+
+	_, err = io.Copy(out, in)
+	if err != nil {
+		return err
+	}
+	return out.Close()
```

Comment:
Since you already have `defer out.Close()` no need to call `out.Close()` at the end

Reply:
Fixed with f75a26712774c479a96a139dc5fdc95d089bb104